src/dms-preview/azext_dms/_help.py

```diff
@@ -111,6 +117,63 @@
                 },
                 ...n
             ]
+        - name: For MongoDB, the format of the database options JSON object.
+          long-summary: |
+            For MongoDB we support per collection migrations. To use this, specify the collections inside the database object as below.
+          text: >
+            {
+                // set to zero to get the default boost during migration (recommended)
+                "boostRUs": 0,
+                // "OneTime" or "Continuous", only "OneTime" is currently supported
+                "replication": "OneTime",
+                // Set to null to use maximum resources available.
+                "throttling": {
+                    // percentage of the CPU to try to avoid using
+                    "minFreeCpu": 25,
+                    // amount of RAM (in MBs) to try to avoid using
+                    "minFreeMemoryMb": 1024,
+                    // max number of collections to migrate at a time
+                    "maxParallelism": 2
+                },
+                "databases": {
```

Comment:
Should this be an array?

Reply:
No. These are implemented as key/value pairs.